### PR TITLE
loot table parser added

### PR DESCRIPTION
**Pull request by DaddyWesker**
Yeah, i know, it looks awful but i haven't found any better solution. 

There are couple TODOs which i don't know we need currently.

Also, there are new type of tool - AnyTool. We need to somehow handle it inside our python scripts. 

To test these changes main of minecraft-demo is enough since I'm using the same function and the same type of output. Though it should contain more entities.

### CODE REVIEW

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java [hunk 1]
```diff
@@ -66,11 +81,203 @@ public void prepare(MissionInit missionInit) {
...
+            triple.add("tool", new JsonPrimitive(tool_name));
+        triple_array.add(triple);
+    }
+
     @Override
     public void writeObservationsToJSON(JsonObject json, MissionInit currentMissionInit) {
         if (!this.sendRec){
             return;
         }
+        Path pth = Path.of("/home/daddywesker/Untitled");
```
Comment (Necr0x0Der) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java]: Is it possible to have a better way to specify path?

Comment (DaddyWesker) [reply src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java]: > Is it possible to have a better way to specify path?

My bad. I've changed that. We dont actually need any path. It is used only to create necessary variables.

**Comment by DaddyWesker** [comment]
I've synced my for with current trueagi-io/Vereya's master.

File: src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java [hunk 2]
```diff
@@ -66,11 +81,203 @@ public void prepare(MissionInit missionInit) {
...
+            triple.add("tool", new JsonPrimitive(tool_name));
+        triple_array.add(triple);
+    }
+
     @Override
     public void writeObservationsToJSON(JsonObject json, MissionInit currentMissionInit) {
         if (!this.sendRec){
             return;
         }
+        Path pth = Path.of("/tmp/");
```
Comment (noskill) [inline src/main/java/io/singularitynet/MissionHandlers/ObservationFromBlocksDropsImplementation.java]: it's safer to use platform independent - System.getProperty("java.io.tmpdir")

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**